### DISCUSSION TITLE
Quaduped and a robotic arm

### DISCUSSION THREAD

**Discussion by yuichiro-student** [google/brax#533] [Q&A]
On the top page of github, I see qudruped and robotic arm moving.
Would you tell me where they are from? This one? 
https://github.com/google-deepmind/mujoco_menagerie/tree/main
I cannot find python scripts for initialization and stepping.

**Comment by btaba** [comment]
@erikfrey

**Comment by trevormcinroe** [comment]
Any update on this?

**Reply by yuichiro-student** [reply]
I think that they are imported from mujoco_menagerie. You still need to write your initialization and stepping functions.
